lisa/tools/wget.py

```diff
@@ -0,0 +1,42 @@
+import pathlib
+from typing import cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Linux
+from lisa.util import LisaException
+
+
+class Wget(Tool):
+    @property
+    def command(self) -> str:
+        return "wget"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def install(self) -> bool:
+        linux_os: Linux = cast(Linux, self.node.os)
+        linux_os.install_packages([self])
+        return self._check_exists()
+
+    def get(
+        self, url: str, file_path: str = "", filename: str = "", overwrite: bool = True
+    ) -> str:
+        # create folder when it doesn't exist
+        self.node.execute(f"mkdir -p {file_path}", shell=True)
+        # get the download file name
+        if not filename:
+            filename = url.rsplit("/", 1)[-1]
+        # combine download file path
+        # TODO: support current lisa folder in pathlib.
+        # So that here can use the corresponding path format.
+        download_path = pathlib.PurePosixPath(f"{file_path}/{filename}")
+        if overwrite:
+            extra_param = " -nc "
+        run_command = f" {url} {extra_param} -O {str(download_path)}"
+        self.run(run_command, no_error_log=True, shell=True)
+        actual_file_path = self.node.execute(f"ls {str(download_path)}", shell=True)
```

Comment:
there is a method called exists on node.shell. But I think we can check exit code of wget to check if the command runs successful.

Reply:
when use -nc, the return code is non-zero

Reply:
If so, can we check stderr? If not, use `shell.exists` is better.